packages/flutter/lib/src/widgets/container.dart

```diff
@@ -310,6 +310,7 @@ class Container extends StatelessWidget {
     this.margin,
     this.transform,
     this.child,
+    this.clipBehaviour,
```

Comment:
I notice in Flutter source code, they use US-English, thus that word is `clipBehavior` instead `clipBehaviour` of UK-English. Should this PR follows the US-English word?